cmd/stack_select.go

```diff
@@ -35,7 +37,8 @@ func newStackSelectCmd() *cobra.Command {
 			"Selecting a stack allows you to use commands like `config`, `preview`, and `update`\n" +
 			"without needing to type the stack name each time.\n" +
 			"\n" +
-			"If no <stack> argument is supplied, you will be prompted to select one interactively.",
+			"If no <stack> argument is supplied, you will be prompted to select one interactively.\n" +
+			"If provided stack name is not found you may pass the -create flag to create and select it",
```

Comment:
I believe this needs to be "pass the --create flag" with two dashes, right?

Reply:
yes, thank you!